src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -511,8 +521,14 @@ private Document parseDocument(URL url) throws IOException, SAXException {
     }
 
     private Document parseDocument(InputStream in) throws SAXException, IOException {
-        try (InputStream ins = in) { // ins will be auto-closed
+        InputStream ins = null;
```

Comment:
Why are we getting rid of try with resources which is the new better way to ensure streams are closed?